File: generated-code-groupA-updated/generated-code-groupA-updated/chatgpt/java-chatgpt-A/Submit_product_review_026_Vulnerable.java
import java.util.ArrayList;
import java.util.List;

public class Submit_product_review_026_detelid {

    static class Review {
        int rating;
        String review;

        Review(int rating, String review) {
            this.rating = rating;
            this.review = review;
        }
    }

    static List<Review> reviews = new ArrayList<>();

    public static String submitReview(
            int rating,
            String review) {

        reviews.add(new Review(rating, review));

        return "Review submitted successfully.";
    }

    public static void main(String[] args) {

        System.out.println(submitReview(
                5,
                "Excellent product!"
        ));
    }
}

File: generated-code-groupA-updated/generated-code-groupA-updated/chatgpt/python-chatgpt-A/Submit_product_review_026_Vulnerable.py
reviews = []


def submit_product_review(rating, review_text):
    """
    Submits a product review.
    """

    reviews.append({
        "rating": rating,
        "review": review_text
    })

    return "Product review submitted successfully."


if __name__ == "__main__":
    print(submit_product_review(
        5,
        "Excellent product!"
    ))